components/FeatureHighlight/src/MDCFeatureHighlightViewController.h

```diff
@@ -150,7 +150,19 @@ typedef void (^MDCFeatureHighlightCompletion)(BOOL accepted);
  Enable legacy font scaling curves for Dynamic Type.
  Default value is NO.
  */
-@property(nonatomic, readwrite, setter=mdc_setLegacyFontScaling:) BOOL mdc_legacyFontScaling;
+@property(nonatomic, readwrite, setter=mdc_setLegacyFontScaling:)
+    BOOL mdc_legacyFontScaling __deprecated;
```

Comment:
Please use `__deprecated_msg("Use blah blah API instead")` to mark deprecated APIs that have replacements.